iptables/iptables.go

```diff
@@ -263,6 +277,29 @@ func (ipt *IPTables) Stats(table, chain string) ([][]string, error) {
 	return rows, nil
 }
 
+// StructuredStats returns statistics as structured data which may be further
+// parsed and marshaled.
+func (ipt *IPTables) StructuredStats(table, chain string) ([]Stat, error) {
+	rawStats, err := ipt.Stats(table, chain)
+	if err != nil {
+		return nil, err
+	}
+
+	structStats := []Stat{}
+	for _, rawStat := range rawStats {
+		if len(rawStat) != 10 {
```

Comment:
Is it possible that this will ever be 11? In other words, could iptables add new stats?

Reply:
I don't think this is likely, but it is possible. 

I can omit this check and instead leave it to the parsing (mentioned below) to handle returning an error if the datatypes are mismatched or the number of fields is not within the expected range.

Reply:
I moved this check into the `ParseStat` method, but made it a minimum of 10 stats in a row to allow for forward compatibility.